EIPS/eip-770.md

```diff
@@ -0,0 +1,124 @@
+## Preamble
+
+    EIP: 770 <or to be assigned by an editor>
+    Title: Decentralized Booking Protocol Standard
+    Author: Vidal Chriqui <vidal@btuprotocol.com>
+    Type: Standard
+    Category : ERC
+    Status: Draft
+    Created: 2017-12-26
+    Requires : ERC 20
+
+## Abstract
+
+The following describes standard functions for a decentralized booking protocol that can be used to reserve any kind of resource (Hotel, concert ticket, restaurant table).
+The protocol involves the following participants :
+
+- Provider : an entity providing its own resource or a third-party party resource to be booked
+- Booker : an entity that books the resource for itself or a third party
+- Booking infrastructure : a public "location" where providers can "post" resource availability data and where these resources can be searched by bookers. The booking infrastructure facilitates signaling between providers and bookers by aggregating a high volume of resource type and availabilities.
+- RES smart contract : A booking smart contract implementation of the decentralized standardized BTU protocol.
+
+## Motivation
+
+The Booking Token Unit (BTU) protocol is a building block for any decentralized application (dApp) or web site willing to implement booking features for their end-users.
```

Comment:
Is the name of the protocol
* The Booking Token Unit protocol, or
* Decentralized Booking Protocol?

If these two are different protocols, where is the Booking Token Unit protocol described?

Reply:
There is only one protocol, the "Booking Token Unit protocol" that is the subject of this ERC

Reply:
The title reads differently.

Reply:
Title was not meant to name the protocol, but thanks for this valuable feedback, I adjusting the title accordingly